### DISCUSSION TITLE
Hello Folks, I want to write a code if a user uploads a PDF and based on its type (2-col or 3-col) the parser will extract data in proper sequence

### DISCUSSION THREAD

**Discussion by DevanshuBrahmbhatt** [jsvine/pdfplumber#976] [Ask for help with specific PDFs]
in this code: I cannot extract properly 2-col data, Please help me.
 page_text = page.extract_text()

Thank you

**Comment by samkit-jain** [comment]
Duplicate of https://github.com/jsvine/pdfplumber/discussions/974
